nengo_gui/viz.py

```diff
@@ -1,419 +1,11 @@
-                self.default_labels[v] = k
-        return config
-
-    def save_config(self, lazy=False, force=False):
-        if not force and not self.config_save_needed:
-            return
-
-        now_time = time.time()
-        if not force and lazy and self.config_save_time is not None:
-            if (now_time - self.config_save_time) < self.config_save_period:
-                return
-
-        with self.lock:
-            self.config_save_time = now_time
-            self.config_save_needed = False
-            try:
-                with open(self.config_name(), 'w') as f:
-                    f.write(self.config.dumps(uids=self.default_labels))
-            except IOError:
-                print("Could not save %s; permission denied" %
-                      self.config_name())
-
-    def modified_config(self):
-        self.config_save_needed = True
-
-    def get_label(self, obj, default_labels=None):
-        if default_labels is None:
-            default_labels = self.default_labels
-        label = obj.label
-        if label is None:
-            label = default_labels.get(obj, None)
-            if label is None:
-                print('ERROR finding label: %s' % obj)
-            else:
-                if '.' in label:
-                    label = label.rsplit('.', 1)[1]
-        if label is None:
-            label = repr(obj)
-        return label
-
-    def get_uid(self, obj, default_labels=None):
-        if default_labels is None:
-            default_labels = self.default_labels
-        uid = default_labels.get(obj, None)
-        if uid is None:
-            uid = repr(obj)
-        return uid
-
-    def start(self, port=8080, browser=True, password=None):
-        """Start the web server"""
-        print("Starting nengo_gui server at http://localhost:%d" % port)
-        if password is not None:
-            nengo_gui.server.Server.add_user('', password)
-            addr = ''
-        else:
-            addr = 'localhost'
-        nengo_gui.server.Server.start(self, port=port, browser=browser, addr=addr)
-
-    def prepare_server(self, viz, port=8080, browser=True):
-        return nengo_gui.server.Server.prepare_server(
-            self, port=port, browser=browser)
-
-    def begin_lifecycle(self, server):
-        nengo_gui.server.Server.begin_lifecycle(
-            server, interactive=self.interactive)
-
-    def create_sim(self):
-        """Create a new Simulator with this configuration"""
-        viz_sim = VizSim(self)
-        self.viz_sims.append(viz_sim)
-        return viz_sim
-
-    def remove_sim(self, viz_sim):
-        self.viz_sims.remove(viz_sim)
-
-    def count_sims(self):
-        return len(self.viz_sims)
+# dummy class so that old scripts still work, but get a deprecation message
+class Viz(nengo_gui.sim_server.SimServer):
+    def __init__(self, *args, **kwargs):
+        warnings.warn("nengo_gui.Viz is deprecated.  "
```

Comment:
This might be controversial? But I think it would be good to get out of the habit of using two spaces between sentences. It's a relic of the typewriter days, and even though this text will be in fixed-width font, it looks really weird to me to see two spaces after a period. I know it's muscle memory, and hard to change, but I think it's worth the week or so of training to conform to modern typography conventions. For my own amusement, I'll post a different [pedantic blog post](http://www.slate.com/articles/technology/technology/2011/01/space_invaders.html) each time I see this in future PRs (unless people strongly think I should let this go) :rainbow: